src/Findologic/Request/Handler/FilterHandler.php

```diff
@@ -235,17 +236,19 @@ private function parseFindologicFiltersForShopware(FiltersExtension $filterExten
 
             if ($filter instanceof RatingFilter) {
                 $max = end($values);
-                $result[$filterName]['max'] = $max->getId();
+                $result[RatingFilter::RATING_FILTER_NAME]['max'] = $max->getId();
             } else {
                 $filterValues = [];
-                foreach ($values as $value) {
-                    $valueId = $value->getUuid() ?? $value->getId();
-                    $filterValues[] = [
-                        'id' => $valueId,
-                        'translated' => ['name' => $value->getTranslated()->getName()]
-                    ];
-
-                    if (!$filter instanceof CategoryFilter) {
+
+                if ($filter instanceof CategoryFilter) {
+                    $this->handleCategoryFilters($values, $filterValues);
+                } else {
+                    foreach ($values as $value) {
+                        $valueId = $value->getUuid() ?? $value->getId();
+                        $filterValues[] = [
+                            'id' => $valueId,
+                            'translated' => ['name' => $value->getTranslated()->getName()]
```

Comment:
This was probably overseen, but the translated value should also contain `$valueId` here.